Codebase viewer responsiveness

BrandonSlade opened the pull request:
The codebase viewer is now fully responsive to the screen size and has labels for the buttons in the side nav bar.  
Edit June 10, 2019 @9:38 AM: The codebase viewer now works in the normal laptop view as well.
<img width="1438" alt="Normal-laptop-view" src="https://user-images.githubusercontent.com/49167261/59199509-2083a680-8b64-11e9-9a2e-5ed52d9a908b.PNG">
<img width="571" alt="IPad-view" src="https://user-images.githubusercontent.com/49167261/59113020-46b20800-8912-11e9-96ea-fd857d4ee617.PNG">
<img width="459" alt="IPhone-5-view" src="https://user-images.githubusercontent.com/49167261/59113021-46b20800-8912-11e9-978b-b8ea5775c9df.PNG">
<img width="1104" alt="Desktop-view" src="https://user-images.githubusercontent.com/49167261/59113022-46b20800-8912-11e9-99c0-0f660b90afc1.PNG">

mgoshorn committed:
Merge pull request #256 from Revature-RPM/responsive_UI_editProject

Responsive ui edit project

BrandonSlade committed:
Brandon Slade June 7, 2019 @9:18 AM

Testing branch but also have some minor changes to the codebase viewer responsiveness.

BrandonSlade committed:
Brandon Slade June 7, 2019 @10:40 AM

Finnished making the codebase viewer Responsive to screen size.
Files changed include zip.component.html, zip.component.scss, and zip.component.ts

mgoshorn reviewed (changes requested):
Overall, I like the way the integration of text into the buttons has gone, but I think it would look better if we could get the sizing of the "home" icon to be similar to that of the "local" icon.

BrandonSlade committed:
Brandon Slade June 10, 2019 @9:38 AM

Fixed the issue with the layout of the codebase viewer when not in mobile view.
